saleor/graphql/wishlist/resolvers.py

```diff
@@ -0,0 +1,20 @@
+from typing import List
+
+from ...wishlist.models import Wishlist, WishlistItem
+from ...wishlist.session_helpers import WishlistSessionHelper
+
+
+def get_wishlist_from_info(info) -> Wishlist:
+    user = info.context.user
+    if user.is_anonymous:
```

Comment:
Wishlists will be basically implemented client-side and only when you log in we will synchronize them with the backend, just as we do with carts/checkouts. One reason for that is that we want to support wishlist in the offline mode which client-side implementation allows.